src/perf/impeller.md

```diff
@@ -0,0 +1,157 @@
+
+```
+  <key>FLTEnableImpeller</key>
+  <true/>
+```
+
+The team continues to make improvements to iOS support,
+and if you encounter any slow frames or rendering
+fidelity issues with Impeller on iOS,
+please file an issue in the [GitHub tracker][]
+with a small reproducible test case, and include
+`[Impeller]` in the title.
+
+[GitHub tracker]: {{site.github}}/flutter/flutter/issues
+
+### Android
+
+Android support is under active development,
+and isn't yet ready for preview. On the master channel,
+it might not work at the tip-of-tree. (QUESTION: What is tip of tree?)
+However, you can get an indication of the direction
+that support is headed by experimenting with it in the
+3.7 stable release. To enable Impeller on Android,
+either pass `--enable-impeller` to the `flutter run`
+command or add the following to your
+`AndroidManifest.xml` file under the `<application>` tag:
+
+```
+  <meta-data
+    android:name="io.flutter.embedding.android.EnableImpeller"
+    android:value="true" />
+```
+
+## Status
+
+You can track the up-to-date status and future plans
+of Impeller by following along on the [GitHub project board][].
+However, here is a summary of recent progress.
+
+[GitHub project board]: {{site.github}}/orgs/flutter/projects/21
+
+The team is increasingly confident that Impeller on iOS
+will meet the rendering needs of nearly all
+existing Flutter apps. As of the 3.7 release,
+Impeller on iOS supports all but a few features.
+The remaining gaps are as follows:
+
+* Setting `Paint.isAntiAlias` to false is not yet implemented
+  ([docs]({{site.api}}/flutter/dart-ui/Paint/isAntiAlias.html)).
+  ([#104721]({{site.github}}/flutter/flutter/issues/104721))
+  ([#109956]({{site.github}}/flutter/flutter/issues/109956))
+* Support for the focal argument to `Gradient.radial`
+  isn't yet implemented ([docs]({{site.api}}/flutter/dart-ui/Gradient/Gradient.radial.html)).
+
+Recently fixed issues: (QUESTION: Should I just delete these?)
+
+* Setting `Paint.invertColors` to true isn't yet implemented
+  ([docs]({{site.api}}/flutter/dart-ui/Paint/invertColors.html)).
+  ([#109739]({{site.github}}/flutter/flutter/issues/109739))
+  ([#113110]({{site.github}}/flutter/flutter/issues/113110))
+* Support for the `blendMode` argument to `Canvas.drawVertices`
+  and `Canvas.drawAtlas` isn't yet implemented
+  ([docs]({{site.api}}/flutter/dart-ui/Canvas/drawVertices.html)).
+  ([#108047]({{site.github}}/flutter/flutter/issues/108047))
+* Support for the `textureCoordinates` argument to the `Vertices`
+  constructor isn't yet implemented
+  [(docs]({{site.api}}/flutter/dart-ui/Vertices-class.html)).
+* Setting `Paint.shader` in text styles isn't yet implemented.
+  ([#120003]({{site.github}}/flutter/flutter/issues/120003))
+
+The team expects to close these gaps over the coming months.
+You might notice minor visual differences in rendering between
+Skia and Impeller.
+These minor differences might (or might not) be bugs,
+so don’t hesitate to file issues.
```

Comment:
Considering link this to https://github.com/flutter/flutter/issues/new/choose so users know where.